data/communities.njk

```diff
@@ -0,0 +1,71 @@
+---
+datasource: 'datasources/communities.yaml'
+permalink: false
+---
+{% set pageTitle = "Comunidades" %}
+{% set navItem = "communities" %}
+
+{% extends "./templates/default.njk" %}
+{% from "./templates/macros/event.njk" import image %}
+
+{% block main %}
+  <div id="svg-icons">
+    <svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
```

Comment:
just wondering, what are the cons and pros of adding the svg directly as paths and not as files imported, as in https://github.com/VLCTechHub/VLCTechHub-site/blob/5cff9d2aecbd5931fa6395934687f40b592f8c2d/templates/past-events.njk#L37 ?

I think they are more reusable when saved as files (and they clutter less the html)

Reply:
Svg's used like images aren't customizables, eg. change color on hover, inline svg's does it possible. I tried to use it as external resource but a first attemp didn't work. I have been googling about svg icons and now it's working as a external resource. The svg file contains the symbol definition and it's only necessary to reference it in inline svg (as an image).

This is currently the preferred solution over icon fonts, [more info](https://www.lambdatest.com/blog/its-2019-lets-end-the-debate-on-icon-fonts-vs-svg-icons/).